src/cli/core/utils/sort.js

```diff
@@ -8,8 +8,8 @@
  * @return {Int}   1 | 0 | -1
  */
 export function byDateDesc(a, b) {
-  var dateA = new Date(a.date)
-  var dateB = new Date(b.date)
+  var dateA = (a.abe_meta.publish != null && a.abe_meta.publish.latest != null) ? new Date(a.abe_meta.publish.latest.date) : 0
```

Comment:
They were not real core functions : we had to do "a.date" to make the sort which relates for sure with the data sent. This kind of function should never know the structure of data sent.